NBISweden/sda-cli
# config  file provided before command

**Related issue(s) and PR(s)**  
This PR closes #442 .


**Description**
The user should be able to provide the config file location argument *ONLY* before the subcommand of the cli. 

**How to test**
For test coverage, some of the existing tests have been altered to follow the new allowed order of providing the config, i.e. before the subcommand.
To manually test, one should test individually the cli commands that are affected. Those are the ones that are using the config as an argument(upload, list htsget, download).
